com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -15,6 +16,28 @@ public enum RenderingMode
         Deferred
     };
 
+    /// <summary>
+    /// When the Universal Renderer should use a depth prepass in Forward mode.
+    /// </summary>
+    public enum DepthPrepassMode
+    {
+        /// <summary>A depth prepass will only be used if needed.</summary>
+        Auto,
+        /// <summary>A depth prepass will always be used.</summary>
+        Forced
+    };
+
+    /// <summary>
+    /// When the Universal Renderer should use a depth prepass in Forward mode.
+    /// </summary>
+    public enum DepthPrimingMode
+    {
+        /// <summary>A depth prepass will only be used if the rendering mode is set to forward and there is a depth prepass used.</summary>
+        Auto,
+        /// <summary>A depth prepass will never be used.</summary>
+        Disabled
+    }
```

Comment:
What's use case that we have depth prepass and want priming mode disabled? Is there any downside to it?

Reply:
I don't think there is a valid use case for it atm. If there is a need for a depth pass it should be derived from the render passes. If there's a reason to have a way to force a depth prepass @Verasl might know. Could be that we both forgot about the prepass being auto injected when we went over this previously.